packages/config/src/memory/memory.config.ts

```diff
@@ -174,7 +174,7 @@ export class ConfigProviderMemory extends BasemapsConfigProvider {
         category: i.category,
         name: targetName,
         format: ImageFormat.Webp,
-        layers: [{ name: targetName, minZoom: 0, maxZoom: 32 }],
+        layers: [{ name: targetName, title: i.title, minZoom: 0, maxZoom: 32 }],
```

Comment:
The All tileset should not be able to be viewed as a big layer, we could set the minZoom: 32?

Reply:
Yes, this already been reset to zoom 32-32 in line 149 when creating all tileset.
This is creating individual tileset so it is zoom 0-32.

Reply:
ahh the github diff showed it as the same block :face_exhaling: